src/braket/ir/jaqcd/shared_models.py

```diff
@@ -129,12 +129,113 @@ class Angle(BaseModel):
     angle: confloat(gt=float("-inf"), lt=float("inf"))
 
 
+class SingleProbability(BaseModel):
+    """
+    A single probability parameter for bit/phase flip noise channel.
+
+    Attributes:
+        probability (float): The probability for noise channel.
+            NaN is not an allowable input.
+
+    Examples:
+        >>> SingleProbability(probability=0.1)
+    """
+
+    probability: confloat(ge=float("0.0"), le=float("0.5"))
+
+
+class SingleProbability_34(BaseModel):
+    """
+    A single probability parameter for depolarizing/two-qubit-dephasing noise channel.
+
+    Attributes:
+        probability (float): The probability for noise channel.
+            NaN is not an allowable input.
+
+    Examples:
+        >>> SingleProbability_34(probability=0.5)
+    """
+
+    probability: confloat(ge=float("0.0"), le=float("0.75"))
+
+
+class SingleProbability_1516(BaseModel):
```

Comment:
No underscores needed in the class names, also the class names should be descriptive; `_34` and `_1516` aren't useful. Also, why isn't `SingleProbability` sufficient?

Reply:
No, Quest has the probability limit for some noise channels, like [0,0.5] for bit_flip and [0,0.75] for depolarizing channel, and we decided to apply that for this launch, so _34 is for channels with limit [0,3/4] and _1516 is for [0,15/16].

Reply:
Worth documenting the 3/4 and 15/16 limits on the angles, and explaining why they exist.

Reply:
sure.